### PR TITLE
Boyd blog post

### PR DESCRIPTION
**Pull request by donboyd5**
@MaxGhenis @jdebacker 

This is my first pull request ever so hope I did it right. The PR includes a .md file with my draft blog post.

A couple of comments:
- When I look at it in my markdown editor (Typora) it looks fine to me.
- When I look at the html file generated by fastpages it looks awful. The bullets don't show up and there seems to be other extraneous information.
- I followed the style guide, which says that every sentence should have its own line. That's not my preference. If it's ok to do it as paragraphs, you'll see that I've put an extra line break to mark where I think paragraphs should be, or I can submit an edited version that has paragraphs.
- It took me forever to figure out what I was supposed to do re fastpages and docker. In the end I think I did what was intended (although the html file does not look right). If you think I did it wrong, please let me know. If you think I did it right, I can offer up some comments that may save other neophytes some of the hair-pulling that I did.

### CODE REVIEW

**Comment by MaxGhenis** [comment]
Thanks @donboyd5. The post looks good to me after running `make server`. The one-line-per-sentence was just for coding to make it easier to review content, multiple sentences per paragraph are fine and I've adjusted it per your note in https://github.com/donboyd5/blog/pull/1. If you can merge that into this branch it'll have to correct date to post tomorrow.

File: settings.ini [hunk 1]
```diff
@@ -7,7 +7,7 @@ description = Writing a library entirely in notebooks
 keywords = jupyter notebook
 author = Sylvain Gugger and Jeremy Howard
 author_email = info@fast.ai
-baseurl = /blog
+baseurl = /fastai-boyd
```
Comment (MaxGhenis) [inline settings.ini:10]: is this necessary?

**Review by MaxGhenis — COMMENTED** [review]
.

**Comment by donboyd5** [comment]
Thanks, @MaxGhenis . Not sure I did this in the right order:
- I merged your commits
- Then changed baseurl in settings.ini to blog. The change to fastai-boyd happened automatically and must be related to how I set up fast pages. I'll have to figure that out so it doesn't happen again in the future.

If this is right we should be good to go. LMK if I need to do anything else.

**Comment by MaxGhenis** [comment]
We're up! http://blog.pslmodels.org/demo-day-14-constructing-tax-data-for-the-50-states

**Comment by jdebacker** [comment]
Thanks for getting this done so quickly @donboyd5 and @MaxGhenis!

**Pull request merged**